### DISCUSSION TITLE
自定义顶栏边缘间距疑问

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1437] [提问 / Q&A] [answered]
当自定义顶栏启用的项目很多时，左边的空白能否自动减少？
现在下图这样，左边一大片空白，右边还得拉滚动条才能看到最右边的栏目是啥，而且还溢出自定义顶栏了，怪别扭的。
![image](https://user-images.githubusercontent.com/49396421/103977025-41cecd80-51b3-11eb-968d-6664b72c7741.png)

**Comment by the1812** [comment, marked as answer]
去顶栏设置减少边缘间距，然后左右各放个弹性空白

**Reply by Geeyun-JY3** [reply]
c忘了可以设置边缘间距了🤦‍♂️发之前没想太多飘了
